spec/system/schools/manual_readings_spec.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
```

Comment:
Could test setup be done in before blocks and expect statements go in 'it' block where possible please?